packages/api/src/server.js

```diff
@@ -42,10 +43,13 @@ app.use(cors());
         },
       },
     }))
-    app.use('/export', (request, response) => {
-      console.log(request)
-      response.send('hello!')
-    })
+
+    if (process.env.READS_DIR) {
+      app.use('/reads', serveStatic(process.env.READS_DIR, {
```

Comment:
Should this default to the resources dir?

Reply:
I don't think so.

The IGV browser requires a reference, which is a 3GB file that I don't think we want to put in the resources directory.
https://github.com/macarthur-lab/gnomad_browser/blob/b4ff2705fd540037acc31f234f660b26162f5207/templates/variant.html#L900-L910

Even if we did include the reference, we wouldn't put all the other read data into the resources folder, so only certain variants would work correctly with resources as the default directory. Such being the case, I think it's preferable to default to not show read data at all in a development environment vs failing to load read data for most variants.